Added new section for CLI tool and updated environment variable [8974] (#184)
eProsima/Fast-RTPS-docs: docs/fastdds/env_vars/env_vars.rst

    +------------------------------------------------------------------+
    | **Linux**                                                        |
    +------------------------------------------------------------------+
    | .. code-block:: bash                                             |
    |                                                                  |
    |    export FASTRTPS_DEFAULT_PROFILES_FILE=/home/user/profiles.xml |
    +------------------------------------------------------------------+
    | **Windows**                                                      |
    +------------------------------------------------------------------+
    | .. code-block:: bash                                             |
    |                                                                  |
    |    set FASTRTPS_DEFAULT_PROFILES_FILE=C:\profiles.xml            |
    +------------------------------------------------------------------+

``ROS_DISCOVERY_SERVER``

>     When this variable is set it configures the participant as a client of

Reviewer: I don't understand this sentence. Please reformulate.

    :ref:`Server-Client Discovery<discovery_server>` provided the
    :ref:`dds_layer_domainParticipant`'s |discoveryProtocol| has been left configured as default.
    The value of the variable must list the locator of the server
    in the form of the IP address (e.g., '192.168.2.23') or IP-port pair (e.g., '192.168.2.23:24353').
    If no port is specified, the default port 11811 is used. For more information on how to configure the discovery

>     mechanism in *Fast DDS*, please refer to ::ref:`discovery`.

Reviewer: ```suggestion
    mechanism in *Fast DDS*, please refer to :ref:`discovery`.
```

    To set more than one address they must be separated by semicolons.
    The server's Id is determined by their position in the list.
    A blank space between semicolons means the corresponding Id is free.

    **Example**

>     The following example shows how to give the address two remote discovery servers with addresses '84.22.259.329:8888'
>     and '81.41.17.102:1234' and Ids 0 and 2.

Reviewer: ```suggestion
    The following example shows how to set the address of two remote discovery servers with addresses '84.22.259.329:8888'
    and '81.41.17.102:1234' and Ids 0 and 2 respectively.
```

    +------------------------------------------------------------------+
    | **Linux**                                                        |
    +------------------------------------------------------------------+
    | .. code-block:: bash                                             |
    |                                                                  |
    |    export FASTRTPS_DEFAULT_PROFILES_FILE=/home/user/profiles.xml |
    +------------------------------------------------------------------+
    | **Windows**                                                      |
    +------------------------------------------------------------------+
    | .. code-block:: bash                                             |
    |                                                                  |
    |    set FASTRTPS_DEFAULT_PROFILES_FILE=C:\profiles.xml            |
    +------------------------------------------------------------------+

``ROS_DISCOVERY_SERVER``

>     Setting this variable configures the participant as a client of
>     :ref:`Server-Client Discovery<discovery_server>` provided its
>     :ref:`dds_layer_domainParticipant`'s |discoveryProtocol| setting has been left configured as default.

Reviewer: The subject is missing. Better in passive form.
DomainPaticipant with a link to its section. 
Simple discovery with a link to its section.

```suggestion
    Setting this variable the DomainParticipant is configured as a Client of the given Server, implementing the 
    :ref:`Server-Client Discovery<discovery_server>` discovery mechanism, provided
    :ref:`dds_layer_domainParticipant`'s |discoveryProtocol| setting has been left configured as default (Simple discovery).
```

>     More information on configuring the DomainParticipant can be found in :ref:`participantprofiles`.

Reviewer: ```suggestion
```

Reviewer: Write a warning note here saying that the environment variable is only used in case the `discoveryProtocol` is set to SIMPLE. In any other case, the environment variable has no effect.
